src/tests/generatekey.cpp

```diff
@@ -812,6 +823,22 @@ TEST_F(rnp_tests, generatekeyECDSA_explicitlySetSmallOutputDigest_DigestAlgAdjus
     rnp_cfg_free(&ops);
 }
 
+TEST_F(rnp_tests, generatekey_multipleUserIds_ShouldFail)
```

Comment:
Please also add a line with test's declaration to rnp_tests.h.
While technically it is not needed it is sometimes handy to have all test names in one header. Or, if not - remove it completely in some separate PR.